library/init/meta/interactive.lean

```diff
@@ -168,11 +168,30 @@ h₂ : b = c
 meta def introv (ns : parse ident_*) : tactic unit :=
 propagate_tags (tactic.introv ns >> return ())
 
+/-- Parse a current name and new name for `rename`. -/
+private meta def rename_arg_parser : parser (name × name) :=
+  prod.mk <$> ident <*> (optional (tk "->") *> ident)
+
+/-- Parse the arguments of `rename`. -/
+private meta def rename_args_parser : parser (list (name × name)) :=
+  (functor.map (λ x, [x]) rename_arg_parser)
+  <|>
+  (tk "[" *> sep_by (tk ",") rename_arg_parser <* tk "]")
+
 /--
-The tactic `rename h₁ h₂` renames hypothesis `h₁` to `h₂` in the current local context.
+Rename one or more local hypotheses. The renamings are given as follows:
+
+```
+rename' x y             -- rename x to y
+rename' x → y           -- ditto
+rename' [x y, a b]      -- rename x to y and a to b
+rename' [x → y, a → b]  -- ditto
```

Comment:
These should be `rename`.

Reply:
Yeah, I just noticed that as well. :( I'll write a doc PR.